xdr/Stellar-transaction.x

```diff
@@ -784,6 +849,8 @@ case opINNER:
         BumpSequenceResult bumpSeqResult;
     case MANAGE_BUY_OFFER:
 	ManageBuyOfferResult manageBuyOfferResult;
+    case PATH_PAYMENT_STRICT_SEND:
+        PathPaymentStrictSendResult pathPaymentStrictSendResult;
```

Comment:
I think there were more changes in XDR files besides this one, see: https://github.com/stellar/go/commit/50b0a8d71a159659a992e72f666900fab365734e#diff-c99bb2d3a1bde548287069d1fe768b58 Can you update all of them?

Reply:
I double checked and all the .x files seem up to date. I think the XDR files in the go mono repo were previously out of date and that's why the go PR include changes to several XDR files.

If you look at https://github.com/stellar/stellar-core/commits/master/src/xdr you will find that the most recent commits to the xdr directory in stellar core involve changes to only Stellar-transaction.x